<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>841</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cooper Black</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>-30</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Runnn!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>131</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProfileButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>131</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="PageTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>315</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NameInputLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>231</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Username or Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordInputLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>210</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>630</y>
     <width>210</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignInButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>630</y>
     <width>210</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="Wrap" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="SearchBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>321</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Search Here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>240</y>
     <width>70</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>770</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contact us via : @comfortablynumb7 on Telegram or 09908611517</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>800</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Company Address : Iran - Tehran - Narmak - IUST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FavoritesButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>30</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Favorites</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CategoriesButton">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>100</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cooper Black</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Categories</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameInput">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>385</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordInput">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>515</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
